contracts/Crowdsale.sol

```diff
@@ -129,7 +129,14 @@ contract Crowdsale is StaffUtil {
 		require(minPurchaseInWei <= maxInvestorContributionInWei);
 		require(address(commissionContract) != address(0));
 	}
-
+	
+	/**
+	function getState
+	
+	Returns information about the current state of the smart contract. 
+	Ex.: paused, finalized, start and end-date, amount of tokens distributed etc.
+	/*
```

Comment:
@mihailozarinschi https://www.bitdegree.org/learn/solidity-syntax#natspec

Reply:
@markholdex with the comments in this format, ending with `/*`, the code just does not compile. I found in the docs that the ending of the comment should be with `*/`. What makes the comments as NatSpec is the fact that they start with `/**` instead of just `/*`